lib/screens/movie_home_page.dart

```diff
@@ -8,14 +8,20 @@ class MovieHomePage extends StatelessWidget {
   const MovieHomePage({super.key});
 
   static const String routeName = "/movie";
+  static const String nullError = "The argument of navigation isn't a movie";
```

Comment:
As a user if a get this message i would not understand whats happening

Reply:
fixed